src/reducers/scrollStateHelper.js

```diff
@@ -437,9 +399,6 @@ export function scrollTo(state, scrollPosition) {
   _updateHeightsInViewport(state, firstRowIndex, firstRowOffset);
   _updateHeightsAboveViewport(state, firstRowIndex);
 
-  _updateRows(state);
-  _recalculateRowHeights(state);
-
   return Object.assign({}, state, {
```

Comment:
The bug was this line was after updateRows. I fix it by refactoring the updateVisibleRows out of these functions